exporter/prometheusremotewriteexporter/config.go

```diff
@@ -32,5 +32,13 @@ type Config struct {
 	// See: https://prometheus.io/docs/practices/naming/#metric-names
 	Namespace string `mapstructure:"namespace"`
 
+	ExternalLabels []ExternalLabel `mapstructure:"external_labels"`
```

Comment:
Why not having a proper map here for the config?

Reply:
i'm not quite sure what you mean, can you clarify?

Reply:
ah I see, so you mean config being
```
prometheusremotewrite:
  external_labels:
    label1: value1
    label2: value2
```

as opposed to my current implementation of 

```
prometheusremotewrite:
  external_labels:
    - key: label1
       value: value1
    - key: label2
       value: value2
```

Looking back now the first is a lot cleaner so I will change it to that